README.md

```diff
@@ -1,87 +1,109 @@
-# Buildkite Test Splitter
+# Buildkite Test Engine Client
 
-Buildkite Test Splitter is an open source tool to orchestrate your test suites. It uses your Buildkite Test Analytics suite data to intelligently partition and parallelise your tests. 
-Test Splitter currently only supports RSpec but support for other frameworks is coming soon.
+Buildkite Test Engine Client (bktec) is an open source tool to orchestrate your test suites. It uses your Buildkite Test Engine suite data to intelligently partition and parallelise your tests.
 
-## Migrating to 0.9.0
-Version 0.9.0 introduces a new feature that requires Test Splitter to read test results from the runner for retries and verification purposes. To enable this feature, it is necessary to configure the `BUILDKITE_SPLITTER_RESULT_PATH` environment variable. This variable specifies the location of where the runner should store test results.
+bktec supports RSpec and Jest.
+
+## Migrating to 1.0.0
+
+The following environment variables are now required
+- `BUILDKITE_TEST_ENGINE_RUNNER`
+  The test runner to use for running tests. You will need to ensure that `BUILDKITE_TEST_ENGINE_TEST_RUNNER` presents in the environment. Currently `rspec` and `jest` are supported.
+- `BUILDKITE_TEST_ENGINE_RESULT_PATH`
+  The location of where the runner should store test results. We introcuded a new feature that requires bktec to read test results from the runner for retries and verification purposes. To enable this feature, it is necessary to configure the `BUILDKITE_TEST_ENGINE_RESULT_PATH` environment variable.
+
+In addition, we have updated the default test command for RSpec to `bundle exec rspec --format progress --format json --out {{resultPath}} {{testExamples}}`. Test Splitter will automatically replace `{{resultPath}}` with the value specified in `BUILDKITE_TEST_ENGINE_RESULT_PATH`. If you want to customize the RSpec command, make sure to include `--format json --out {{resultPath}}` in the command. 
```

Comment:
```suggestion
- `BUILDKITE_TEST_ENGINE_RUNNER`
 
  The test runner to use for running tests. You will need to ensure that `BUILDKITE_TEST_ENGINE_TEST_RUNNER` presents in the environment. Currently `rspec` and `jest` are supported.
  
- `BUILDKITE_TEST_ENGINE_RESULT_PATH`
  
  The location of where the runner should store test results. We introcuded a new feature that requires bktec to read test results from the runner for retries and verification purposes. To enable this feature, it is necessary to configure the `BUILDKITE_TEST_ENGINE_RESULT_PATH` environment variable.

  In addition, we have updated the default test command for RSpec to `bundle exec rspec --format progress --format json --out {{resultPath}} {{testExamples}}`. Test Splitter will automatically replace `{{resultPath}}` with the value specified in `BUILDKITE_TEST_ENGINE_RESULT_PATH`. If you want to customize the RSpec command, make sure to include `--format json --out {{resultPath}}` in the command. 
```

Reply:
@nprizal your suggestion has a typo in `We introcuded a new`

Reply:
Yeah I noticed that, I've fixed it in my last commit